src/open_producten/logging/serializing.py

```diff
@@ -17,10 +19,18 @@ def model_to_dict(instance):
         match f:
             case models.ManyToManyField():
                 value = [obj.pk for obj in value]
+            case models.FileField():
+                value = value.name
             case _:
                 pass
         data[f.name] = value
 
+    for obj in opts.related_objects:
+        manager = getattr(instance, obj.related_name, None)
+        data[obj.related_name] = [
+            model_to_dict(instance) for instance in manager.iterator()
+        ]
```

Comment:
If `manager` is `None`, doing `manager.iterator()` will probably raise an AttributeError, right?

Reply:
added a check